tests/lib/test_archive.py

```diff
@@ -121,6 +153,19 @@ def test_unpack_tar(tar: pathlib.Path, tmp_path: pathlib.Path) -> None:
     assert dest.joinpath("hello.txt").read_text() == "hello world\n"
 
 
+def test_unpack_tar_strip1(tar: pathlib.Path, tmp_path: pathlib.Path) -> None:
+    dest = tmp_path.joinpath("dest")
+    archive.unpack(
```

Comment:
erm this should be raising since there's no component to strip, but i need to go eat lunch soon